targets/XPlatCppSdk/source/build/Android/XPlatCppSdk/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.microsoft.xplatcppsdk" >
+    package="com.playfab.xplatcppsdk" >
```

Comment:
Yes, I agree. We don't want to confuse the issue right now.
PlayFab SDK's should stay com.playfab.etc until we eventually make a single big move to make it universally com.microsoft.playfab.etc or whatever.
But until then, we should avoid the confusion and stay com.playfab.

Reply:
There's a more pragmatic reason too- the test assemblies all look for "com.playfab.service" when running on AppCenter. :)  All our apps need the same package name otherwise I would have to do some uggo matching code in the test assemblies.  yuck!

Reply:
Sounds awesome.